content/pages/community.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Community</title>
+        <meta name="save_as" content="community/index.html">
+        <meta name="url" content="community/">
+        <meta name="body_class" content="community_page">
```

Comment:
As currently defined, you don't need this class because it is not used. Remove it.